code/modules/admin/verbs/secrets.dm

```diff
@@ -604,6 +604,24 @@ ADMIN_VERB(secrets, R_NONE, "Secrets", "Abuse harder than you ever have before w
 			message_admins("[key_name_admin(holder)] [ctf_controller.instagib_mode ? "enabled" : "disabled"] instagib mode in CTF game: [selected_game]")
 			log_admin("[key_name_admin(holder)] [ctf_controller.instagib_mode ? "enabled" : "disabled"] instagib mode in CTF game: [selected_game]")
 
+		if("mass_heal")
+			if(!is_funmin)
+				return
+			var/heal_mobs = tgui_alert(usr, "Heal all mobs and return ghosts to their bodies?", "Mass Healing", list("Yes", "No", "Cancel"))
+			if(!heal_mobs || heal_mobs == "Cancel")
```

Comment:
You offer three options, "Yes" "No" "Cancel"

...But "No" doesn't do anything

Reply:
![chrome_8yDTSCDl0f](https://github.com/tgstation/tgstation/assets/4081722/504cb3bc-28ce-47bc-aaa7-c4ea78fae59d)